tce/src/bintools/BlocksTranslator/CgraAlu.cc

```diff
@@ -0,0 +1,116 @@
+/**
+ * @file CgraAlu.cc
```

Comment:
Can you rename the abbreviations to capitals in filenames and type names, to match the other TCE style? Here BlocksALU.cc might be more descriptive name? Also in general you might want to avoid using CGRA but Blocks to be more specific.

Reply:
Updated naming to Blocks!